package.json

```diff
@@ -25,7 +25,6 @@
     "list": "lerna list",
     "new-component": "node ./scripts/new.js",
     "postinstall": "husky install",
-    "prepublishOnly": "yarn run build",
```

Comment:
`build` already happens in the release GHA flow https://github.com/nylas/components/blob/d0b1e8ecdbcaba875f1b03fc3a580569bb06ba20/.github/workflows/build-test-publish.yaml#L129